fdbserver/storageserver.actor.cpp

```diff
@@ -1985,6 +1993,38 @@ void merge(Arena& arena,
 	}
 }
 
+ACTOR Future<Optional<Value>> quickGetValue(StorageServer* data, StringRef key, Version version) {
+	if (data->shards[key]->isReadable()) {
+		try {
+			// TODO: Use a lower level API may be better? Or tweak priorities?
+			GetValueRequest req(Span().context, key, version, Optional<TagSet>(), Optional<UID>());
+			data->actors.add(data->readGuard(req, getValueQ));
+			GetValueReply reply = wait(req.reply.getFuture());
+			//			std::cout << "quickGetValue hit local shard" << std::endl;
+			++data->counters.quickGetValueHit;
+			return reply.value;
+		} catch (Error& e) {
+			//			std::cout << "quickGetValue fallback because of exception " << e.name() << std::endl;
```

Comment:
I would use TraceEvent() than std::cout.

Reply:
Removed. I already have counters for logging these infomation